tests/testData/mockData.ts

```diff
@@ -1837,19 +1868,22 @@ export const expectedPeriodMultipleServicesTicketWithMultipleProductsAndMultiple
             nocCode: 'WBTR',
             selectedServices: [
                 {
-                    lineName: '237',
-                    serviceCode: '11-237-_-y08-1',
-                    serviceDescription: 'Ashton Under Lyne - Glossop',
+                    lineName: '343',
```

Comment:
I'd be tempted to keep the services for each additional operator a bit different (maybe just a different lineName / lineId)